src/Util/FilterUtil.ts

```diff
@@ -117,7 +117,7 @@ class FilterUtil {
     const prop: any = _get(feature, 'properties[' + filter[1] + ']');
     switch (filter[0]) {
       case '==':
-        return (prop === filter[2]);
+        return (('' + prop) === ('' + filter[2]));
```

Comment:
Nice approach but i'm afraid we won't fit any cases of string-number comparsion... (don't know if that is what we want)

e.g When `featureValue` is `'2.00'` and `filterValue` is `2`:

```javascript
'' + '2.00' === '' + 2 // false
```

Reply:
True, but then again the previous approach would also lead to `false` here.